did/icrc1_auction.did

```diff
@@ -100,7 +100,15 @@ type PlaceArg = vec variant {
 };
 
 type ManageOrdersResponse = variant {
-  Ok : vec OrderId;
+  Ok : record {
+    vec record {
+      OrderId;
+      Token;
+      nat;
+      float64;
+    };
+    vec OrderId;
+  };
```

Comment:
Is this type chosen to be backwards compatible in the interface?

Because each OrderId is present twice here isn't it?